src/main/java/bio/terra/validation/Utils.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.validation;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public final class Utils {
+
+    private static final String VALID_NAME_REGEX = "[a-zA-Z0-9_]{1,63}";
+
+    private Utils() {}
+
+    public static <T> boolean hasDuplicates(List<T> list) {
+        HashSet<T> seen = new HashSet<>();
```

Comment:
you could create a set from the list and then check if they have the same number of items. there is also a way to use .stream().distinct().collect(...) and then compare the sizes

return !list.size().equals(new HashSet(list).size();